src/node_api.cc

```diff
@@ -741,8 +741,19 @@ napi_status napi_define_class(napi_env env,
     v8::Local<v8::String> property_name;
     CHECK_NEW_FROM_UTF8(isolate, property_name, p->utf8name);
 
+    // convert the properties from NAPI to v8 format
+    unsigned int attribute_flags = v8::None;
```

Comment:
Is it possible to declare the variable as `v8::PropertyAttribute` so the cast below wouldn't be necessary?